index.html

```diff
@@ -309,9 +310,12 @@ <h1 class="h1 regular"><a class="underline" href="{{ post.url }}">{{ post.title
                 </div>
                 {% endfor %}
                 <h1 class="h1 regular">Older Announcements</h2>
-                  {% for post in site.posts offset:1 %}
+                  {% for post in site.posts offset:1 limit:5 %}
```

Comment:
I think we should have at least 10
```suggestion
                  {% for post in site.posts offset:1 limit:10 %}
```

Reply:
5 currently limits it to posts since 2018, I don't think we really need to go back further than that on the home page.

Reply:
But we will need to change that very soon in future - like we'll have posts enough soon enough. 5 feels to empty.